rocketpy/utilities.py

```diff
@@ -197,3 +196,46 @@ def du(z, u):
         velocityFunction()
 
     return altitudeFunction, velocityFunction, final_sol
+
+
+def create_dispersion_dictionary(filename):
+    """Creates a dictionary with the rocket data provided by a .csv file.
+    File should be organized in four columns: attribute_class, parameter_name,
+    mean_value, standard_deviation. The first row should be the header.
+    It is advised to use ";" as separator, but "," should work on most of cases.
+
+    Parameters
+    ----------
+    filename : string
+        String with the path to the .csv file.
+
+    Returns
+    -------
+    dictionary
+        Dictionary with all rocket data to be used in dispersion analysis.

```

Comment:
Please describe the "format" of this dictionary. Consider adding examples of when this utility can be useful in the docstring.